query/query.go

```diff
@@ -107,28 +107,37 @@ type Latency struct {
 
 // params contains the list of parameters required to execute a SubGraph.
 type params struct {
```

Comment:
I don't want to make `params` an exported type because it's not being used outside of the query package. I went for making all fields start with an uppercase because there were just a few that broke this convention.